wheel_matrix.yml

```diff
@@ -1,4 +1,44 @@
 packages:
+  bcrypt:
+    versions:
+      latest:
+        wheels:
+        - platform_tag: freebsd_12_2_release_amd64
+          platform_instance: freebsd/12.2
+          platform_arch: x86_64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_12_2_release_arm64
+          platform_instance: freebsd/12.2
+          platform_arch: aarch64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_13_0_release_amd64
+          platform_instance: freebsd/13.0
+          platform_arch: x86_64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_13_0_release_arm64
+          platform_instance: freebsd/13.0
+          platform_arch: aarch64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_13_1_release_amd64
+          platform_instance: freebsd/13.1
+          platform_arch: x86_64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_13_1_release_arm64
+          platform_instance: freebsd/13.1
+          platform_arch: aarch64
+          python:
+          - tag: cp38
+            abi: abi3
```

Comment:
The other packages list all the x86_64 builds before the aarch64 builds. It would be nice to maintain consistency by doing the same here.

Reply:
done